ssh/tailssh/incubator.go

```diff
@@ -310,15 +310,22 @@ func (ss *sshSession) launchProcess() error {
 	if err != nil {
 		return err
 	}
-	go resizeWindow(pty, winCh)
-	ss.stdout = pty // no stderr for a pty
-	ss.stdin = pty
+	// We need to handle closing the the stdin and stdout separately from each other;
+	// to do this we dup the pty FD and assign one to stdin and one to stdout.
+	ptyDup, err := syscall.Dup(int(pty.Fd()))
+	if err != nil {
+		return err
+	}
+	go resizeWindow(ptyDup, winCh)
+	ss.stdout = os.NewFile(uintptr(ptyDup), pty.Name())
+	ss.stdin = pty // no stderr for a pty
```

Comment:
why's comment on this line now? made more sense by stdout before.

Maybe `ss.stderr = nil // none for a pty` ?